src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -398,13 +400,23 @@ public static final class Config extends AbstractBigQueryActionConfig {
     @Description("Timeout in seconds to read data from an established HTTP connection (Default value is 120).")
     private Integer readTimeout;
 
+    @Name(WRITE_PREFERENCE)
+    @Nullable
+    @Macro
+    @Description("Specifies the write preference for the destination table. Options are: " +
+        "Write if Empty: Only write if the table is empty. " +
+        "Append to Table: Add results to existing data. " +
+        "Overwrite Table: Replace all existing data.")
```

Comment:
Description/Tooltip should be crisp: `Specifies whether a job should overwrite or append the existing destination table if it already exists.`

Please add these details in markdown: https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryExecute-action.md